src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
```

Comment:
The default compare function should return a number (negative, zero, positive), but `String(a) > String(b)` returns a boolean. Consider using localeCompare or a numerical comparison for proper sorting.